docs/advanced.rst

```diff
@@ -281,6 +281,7 @@ flag::
     pytest==3.2.3
 
 Adding the ``--hash`` flag will add package hashes to the output for extra security.
+Adding the ``--exclude-markers`` flagwill exclude the markers from the output.
```

Comment:
typo need space `flag will`  -- I tried to edit this but I don't think I can directly without some hoop jumping.